Source/bmalloc/libpas/src/libpas/pas_thread_local_cache.h

```diff
@@ -118,9 +126,13 @@ static inline bool pas_thread_local_cache_can_set(void)
 {
 #ifdef PAS_THREAD_LOCAL_CACHE_CAN_DETECT_THREAD_EXIT
     return !pthread_self_is_exiting_np();
+#else
+#if PAS_HAVE_THREAD_KEYWORD
+    return ((uintptr_t)pas_thread_local_cache_try_get_impl()) >= PAS_THREAD_LOCAL_CACHE_UNINITIALIZED;
 #else
     return ((uintptr_t)pas_thread_local_cache_try_get_impl()) != PAS_THREAD_LOCAL_CACHE_DESTROYED;
```

Comment:
...still there's an implementation for non PAS_HAVE_THREAD_KEYWORD case.

Reply:
Older internal macOS does not have `PAS_THREAD_LOCAL_CACHE_CAN_DETECT_THREAD_EXIT`. So we need to keep it regardless of whether `PAS_HAVE_THREAD_KEYWORD` exists.